### DISCUSSION TITLE
how to get a plain output

### DISCUSSION THREAD

**Discussion by electriquo** [TomWright/dasel#283] [Q&A] [answered]
Dasel v1 supports the `--plain` option. Is there any equivalent support in Dasel v2?

```shell
$ dasel --version
dasel version 2.0.2

$ echo '{"tags": ["foo","bar"]}' | dasel -r json '.tags.all()'
"foo"
"bar"
```

How can I make Dasel return the values without the quotes? For instance, if `--plain` was supported in Dasel v2

```shell
$ echo '{"tags": ["foo","bar"]}' | dasel -r json '.tags.all()' --plain
foo
bar
```

Tried to utilize the csv output to get the plain values, but got an error

```shell
$ echo '{"tags": ["foo","bar"]}' | dasel -r json '.tags.all()' -w csv
Error: CSVParser.toBytes cannot handle type string
```

Switching to yml output seems to do well for some values (but observe the inconsistency in the output)

```shell
$ echo '{"tags": ["foo","bar", "1.1", "*"]}' | dasel -r json '.tags.all()' -w yml
foo
bar
"1.1"
'*'
```

If there is a better selector for getting all values in plain form, please enlighten me :)

**Comment by TomWright** [comment, marked as answer]
That should be `-w -`. Looks like I missed it from the docs... Sorry about that

Your CSV problem is because of the `all()` selector.

You're passing a list of strings with no context to the csv parser.
The csv parser expects an array. You can achieve this by using `merge` or removing `all`.

**Reply by electriquo** [reply]
Thank you.
Would you like me to open a pull-request for the docs?

**Reply by TomWright** [reply]
No problem.

That's OK I can get it done tomorrow 👍

**Reply by TomWright** [reply]
I've made a small update here: https://daseldocs.tomwright.me/supported-file-formats
